_episodes_rmd/04-data-structures-part1.Rmd

```diff
@@ -227,7 +228,7 @@ empty character strings. If we similarly do
 str(cats$weight)
 ```
 
-we see that that's a vector, too - *the columns of data we load into R
+we see that is a vector, too - *the columns of data we load into R
```

Comment:
I would change to "we see that `cats$weight` is a vector, too" to clarify further. This also helps to avoid the double "that".